indigo_app/templates/place/detail.html

```diff
@@ -8,10 +8,10 @@
 
   <div class="row">
     <div class="col">
-      <div class="card mt-3">
+      <div class="card mt-4">
         <div class="card-header">
           <a class="float-right" href="{% url 'tasks' place=place.place_code %}">Tasks →</a>
-          <h5 class="mb-0">Open tasks</h5>
+          <h5 class="mb-0">Current tasks ({{ place.place_tasks.unclosed.count }})</h5>
```

Comment:
@ro6ley I think you already have a variable with this number?

Reply:
Yes, I have the number